### DISCUSSION TITLE
Occupancy ‘stuck ‘

### DISCUSSION THREAD

**Discussion by pocipocster** [blakeblackshear/frigate#9055] [General]
Not sure what’s happening here.
Though recognising the sofa as a person would assume as ‘stationary’ would eventually clear - but did not .

![IMG_9062](https://github.com/blakeblackshear/frigate/assets/67703825/e9dfe7e0-f879-49a3-a375-24c47142cacd)

**Comment by NickM-27** [comment]
If there's no motion in 0.12 then by default the object is considered there until motion occurs. 

In frigate 0.13 objects are continuously verified by default (this can be configured in 0.12)

**Comment by pocipocster** [comment]
Thanks ! Upgraded to 0.13 . Will try .
Much appreciated!! 👍
